spec/controllers/brapi/v1/studies_controller_spec.rb

```diff
@@ -0,0 +1,167 @@
+require 'rails_helper'
+require 'pp'
+
+RSpec.describe Brapi::V1::StudiesController do
+  
+  context 'BrAPI without requiring authentication' do
+    let(:u1) { create :user }
+    before { sign_in u1 }
+    
+    describe '#search' do
```

Comment:
I've created some of them in last commit based on previous controller specs. Do I remove these controller ones?